package/Dockerfile

```diff
@@ -14,6 +14,7 @@ WORKDIR /var/lib/rancher
 ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
 ARG SYSTEM_CHART_DEFAULT_BRANCH=v2.3-dev
+ARG RANCHER_METADATA_BRANCH=
```

Comment:
since the default is assumed "", we should comment that the logic of picking dev vs master is handled somewhere else and that this is for if an explicit branch (besides dev and master) need to be used.

Reply:
updated with comment